### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by guahanweb**
See the commit comment for the best description of these changes. This is all simplification and further modularization of code paths.

### CODE REVIEW

**Comment by guahanweb** [comment]
Clean test run:
image

**Comment by acabunoc** [comment]
Looking through this, I think this includes changes from #88. Makes sense to deal with #88 before looking at this one :)

**Comment by guahanweb** [comment]
Linting passes:
image

Tests passing:
image

File: test/helpers.js [hunk 1]
```diff
@@ -15,34 +24,19 @@ describe('helpers', function () {
...
   it('modEntry deletes only email from entry', function () {
-    var entry = {
-      email: 'test@orcid.org',
-      science: 'lab'
-    };
-    helpers.modEntry(entry, 'test');
-    assert.deepEqual(entry, {
-      orcid: 'test',
+    entry.science = 'lab';
+    assert.deepEqual(helpers.modEntry(entry), {
```
Comment (acabunoc) [inline test/helpers.js]: Yes, THANK YOU for this. This addresses #66 nicely :)

**Comment by acabunoc** [comment]
This restructure makes a lot of sense to me and builds on @aid29's work nicely :) R+ for me! Merging.

**Pull request merged**